package.json

```diff
@@ -65,7 +65,7 @@
         "eslint-plugin-import": "^2.29.1",
         "eslint-plugin-jest": "^27.9.0",
         "husky": "^9.0.11",
-        "jest": "^29.7.0",
+        "jest": "30.0.0-alpha.1",
```

Comment:
in order to be able to use `matchInlineSnapshot` matchers
https://github.com/jestjs/jest/pull/14566

Reply:
How unstable is it? `alpha.1` sounds scary :D

Reply:
none of the existing tests broke, so I it looks ok I'd say.
Looking at the [breaking changes](https://github.com/jestjs/jest/releases/tag/v30.0.0-alpha.1), it does not look like impacting how zk-kit uses jest.
There is `alpha.3` released since.
As it is "only" a devDep, I'd YOLO merge it ^^.
Your call. I can also stop using the snapshot matchers in the tests I added too...

Reply:
I'm in, you convinced me ahah 👍🏽